### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jjasghar**
Added Important to make the Mac location more obvious.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,7 +366,8 @@ After running `ilab config init` your directories will look like the following o
  3. `~/.local/share/instructlab/taxonomy/`: Contains the skill and knowledge data.
  4. `~/.local/share/instructlab/checkpoints/`: Contains the output of the training process
 
- On MacOS, these directories will be under `Library/Application Support/instructlab`. This directory setup is temporary in 0.18.0 and will mimic the Linux paths in future releases. The models directory will be under `Library/Caches/instructlab`.
+> [!IMPORTANT]
+> On MacOS, these directories will be under `Library/Application Support/instructlab`. This directory setup is temporary in 0.18.0 and will mimic the Linux paths in future releases. The models directory will be under `Library/Caches/instructlab`.
```
Comment (bjhargrave) [inline README.md:370]:
```suggestion
> On MacOS, these directories will be under `~/Library/Application Support/instructlab`. This directory setup is temporary in 0.18.0 and will mimic the Linux paths in future releases. The models directory will be under `~/Library/Caches/instructlab`.
```

**Review by bjhargrave — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -366,7 +366,8 @@ After running `ilab config init` your directories will look like the following o
  3. `~/.local/share/instructlab/taxonomy/`: Contains the skill and knowledge data.
  4. `~/.local/share/instructlab/checkpoints/`: Contains the output of the training process
 
- On MacOS, these directories will be under `Library/Application Support/instructlab`. This directory setup is temporary in 0.18.0 and will mimic the Linux paths in future releases. The models directory will be under `Library/Caches/instructlab`.
+> [!IMPORTANT]
+> On MacOS, these directories will be under `~/Library/Application Support/instructlab`. This directory setup is temporary in 0.18.0 and will mimic the Linux paths in future releases. The models directory will be under `~/Library/Caches/instructlab`.
```
Comment (nathan-weinberg) [inline README.md:370]:
```suggestion
> On MacOS, these directories will be under `~/Library/Application Support/instructlab`. This directory setup is temporary in 0.18.z and will mimic the Linux paths in future releases. The models directory will be under `~/Library/Caches/instructlab`. For more information see https://github.com/instructlab/instructlab/issues/1747.
```

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

**Comment by nathan-weinberg** [comment]
Please also squash commits/update commit and PR desc to be specific to what is being done here - thank you!

File: README.md (hunk 2 above)
Comment (kelbrown20) [reply README.md:370]: +1 to this suggestion

**Review by kelbrown20 — COMMENTED** [review]
Just the rebase and @nathan-weinberg suggestion, otherwise looks great!

File: README.md (hunk 2 above)
Comment (bjhargrave) [reply README.md:370]: We can remove this note entirely since 0.19 is out and uses the XDG folders like linux.

**Comment by nathan-weinberg** [comment]
This is seemingly abandoned and the directory structure has changed, closing this

**Pull request closed without merging**